Source/WebKit/Shared/WebWheelEvent.cpp

```diff
@@ -78,6 +78,12 @@ WebWheelEvent::WebWheelEvent(WebEvent&& event, const IntPoint& position, const I
 }
 #endif
 
+WebWheelEvent WebWheelEvent::isolatedCopy() const
```

Comment:
Looking at this closer, it doesn't seem like any part of `WebWheelEvent` actually requires `isolatedCopy`-ing (for instance, any strings). I think we can just do one of the following:
- Rename this new method to just `copy()`.
- Use `WebWheelEvent { … }` in the lambda capture below to force a copy instead of using this new method.